database/pgsql/layer.go

```diff
@@ -235,12 +235,13 @@ func (tx *pgSession) persistLayerFeatures(features []dbLayerFeature) error {
 	sort.Slice(features, func(i, j int) bool {
 		return features[i].featureID < features[j].featureID
 	})
-
-	keys := make([]interface{}, len(features)*3)
-	for i, feature := range features {
-		keys[i*3] = feature.layerID
-		keys[i*3+1] = feature.featureID
-		keys[i*3+2] = feature.detectorID
+	var keys []interface{}
+	// keys := make([]interface{}, len(features)*3)
```

Comment:
why comment this out? this is probably what we want to keep rather than `var keys []interface{}`, which introduces the possibility of a nil slice.